### PR TITLE
Adding SDK examples

### PR DESCRIPTION
**Pull request by wonderlandpark**

### CODE REVIEW

**Comment by wonderlandpark** [comment]
@SaidBySolo @zero734kr Could you add each SDK's examples?

**Comment by SaidBySolo** [comment]
> @SaidBySolo @zero734kr Could you add each SDK's examples?

 https://github.com/koreanbots/py-sdk doesn't yet support an endpoint to get votes.

PR is open.

File: docs/예시/유저 투표 여부 확인하기.md [hunk 1]
```diff
@@ -2,4 +2,12 @@
 
 해당 페이지에서는 유저의 투표 여부를 확인하는 코드를 포함하고 있습니다.
 
-## SDK를 사용하여 재작성 예정
+## Python
+
+```py
+```
```
Comment (SaidBySolo) [inline docs/예시/유저 투표 여부 확인하기.md]:
```suggestion

```py
import koreanbots
import asyncio

kb = koreanbots.Koreanbots(api_key="api_key")

data = asyncio.run(kb.is_voted(285185716240252929, 653534001742741552))

print(data.voted)

```

```

File: docs/예시/봇 서버 수 업데이트하기.md [hunk 2]
```diff
@@ -2,4 +2,12 @@
 
 해당 페이지에서는 봇의 서버 수를 확인하는 코드를 포함하고 있습니다.
 
-## SDK를 사용하여 재작성 예정
+## Python
+
+```py
+```
```
Comment (SaidBySolo) [inline docs/예시/봇 서버 수 업데이트하기.md]:
```suggestion
```py
import koreanbots
import asyncio
kb = koreanbots.Koreanbots()
data = asyncio.run(kb.botinfo(653534001742741552))
print(data.servers)

```
```

**Review by SaidBySolo — CHANGES_REQUESTED** [review]

File: docs/예시/봇 서버 수 업데이트하기.md [hunk 3]
```diff
@@ -2,4 +2,12 @@
...
+
+## Javascript
+
+```js
```
Comment (zero734kr) [inline docs/예시/봇 서버 수 업데이트하기.md:21]:
```suggestion
```js
const { Koreanbots } = require("koreanbots")
const koreanbots = new Koreanbots({
    api: {
        token: "KOREANBOTS 토큰"
    },
    clientID: "봇 아이디"
})

koreanbots.mybot.update({ servers: 서버수 })
    .then(console.log)
```

File: docs/예시/봇 서버 수 업데이트하기.md [hunk 4]
```diff
@@ -2,4 +2,12 @@
...
 해당 페이지에서는 봇의 서버 수를 확인하는 코드를 포함하고 있습니다.
 
-## SDK를 사용하여 재작성 예정
+## Python
+
+```py
+```
+
+## Javascript
+
```
Comment (zero734kr) [inline docs/예시/봇 서버 수 업데이트하기.md:18]:
```suggestion

> https://js-sdk-docs.pages.dev/classes/BotManager

```

File: docs/예시/유저 투표 여부 확인하기.md [hunk 5]
```diff
@@ -2,4 +2,12 @@
...
+
+## Javascript
+
+```js
```
Comment (zero734kr) [inline docs/예시/유저 투표 여부 확인하기.md:23]:
```suggestion
```js
const { Koreanbots } = require("koreanbots")
const koreanbots = new Koreanbots({
    api: {
        token: "KOREANBOTS 토큰"
    },
    clientID: "봇 아이디"
})

koreanbots.mybot.checkVote("유저 아이디")
    .then(console.log)
```

File: docs/예시/유저 투표 여부 확인하기.md [hunk 6]
```diff
@@ -2,4 +2,12 @@
...
 해당 페이지에서는 유저의 투표 여부를 확인하는 코드를 포함하고 있습니다.
 
-## SDK를 사용하여 재작성 예정
+## Python
+
+```py
+```
+
+## Javascript
+
```
Comment (zero734kr) [inline docs/예시/유저 투표 여부 확인하기.md:20]:
```suggestion

> https://js-sdk-docs.pages.dev/classes/Mybot

```

**Review by zero734kr — CHANGES_REQUESTED** [review]

File: docs/예시/봇 서버 수 업데이트하기.md (hunk 2 above)
Comment (wonderlandpark) [reply docs/예시/봇 서버 수 업데이트하기.md]: The description was wrong. Could you check it again?

File: docs/예시/봇 서버 수 업데이트하기.md (hunk 3 above)
Comment (wonderlandpark) [reply docs/예시/봇 서버 수 업데이트하기.md:21]: The description was wrong. Could you check it again?

Comment (zero734kr) [reply docs/예시/봇 서버 수 업데이트하기.md:21]: @wonderlandpark

Comment (wonderlandpark) [reply docs/예시/봇 서버 수 업데이트하기.md:21]: Could you change `count` to `servers`

Comment (zero734kr) [reply docs/예시/봇 서버 수 업데이트하기.md:21]: @wonderlandpark

**Review by zero734kr — APPROVED** [review]

File: docs/예시/봇 서버 수 업데이트하기.md [hunk 7]
```diff
@@ -1,5 +1,27 @@
...
 
-해당 페이지에서는 봇의 서버 수를 확인하는 코드를 포함하고 있습니다.
+해당 페이지에서는 봇의 서버 수를 업데이트하는 코드를 포함하고 있습니다.
 
-## SDK를 사용하여 재작성 예정
+## Python
+
+```py
+
+```
```
Comment (SaidBySolo) [inline docs/예시/봇 서버 수 업데이트하기.md]:
```suggestion
```py

from discord import Client
from koreanbots.client import Koreanbots

client = Client()
kbots = Koreanbots(api_key="API_KEY")
await kbots.guildcount(client.user.id, servers=len(client.guilds))
```
```

File: docs/예시/유저 투표 여부 확인하기.md [hunk 8]
```diff
@@ -2,4 +2,33 @@
...
+```py
+import koreanbots
+import asyncio
+
+kb = koreanbots.Koreanbots(api_key="api_key")
+
+data = asyncio.run(kb.is_voted(285185716240252929, 653534001742741552))
+
+print(data.voted)
+```
```
Comment (SaidBySolo) [inline docs/예시/유저 투표 여부 확인하기.md]:
```suggestion

```py
import koreanbots
import asyncio

kbots = koreanbots.Koreanbots(api_key="api_key")

data = asyncio.run(kbots.is_voted(285185716240252929, 653534001742741552))

print(data.voted)
```

```

**Review by SaidBySolo — CHANGES_REQUESTED** [review]

**Review by SaidBySolo — CHANGES_REQUESTED** [review]
Cheak markdown

**Pull request merged**